examples/places-autocomplete/components/places-search/places-search.tsx

```diff
@@ -35,24 +33,25 @@ const PlacesSearch = () => {
     setInputValue(event.target.value);
   };
 
+  // Add a marker whenever a place was selected
   useEffect(() => {
-    if (map && selectedPlace) {
-      if (marker) {
-        marker.setMap(null);
-      }
+    if (!map || !selectedPlace) {
+      return () => {};
+    }
 
-      const markerOptions: google.maps.MarkerOptions = {
-        map,
-        position: selectedPlace?.geometry?.location,
-        title: selectedPlace.name,
-        clickable: false
-      };
+    const markerOptions: google.maps.MarkerOptions = {
+      map,
+      position: selectedPlace?.geometry?.location,
```

Comment:
```suggestion
      position: selectedPlace.geometry?.location,
```
This should work now, no?